data/mods/More_Survival_Tools/item_groups.json

```diff
@@ -0,0 +1,226 @@
+[
```

Comment:
Addition of withered plants to forage itemgroups can be yanked out at your discretion, but keeping it as mod-only works too. Only important thing here in the forage entries is adding garlic.